ide/html.editor/test/unit/src/org/netbeans/modules/html/editor/completion/HtmlCompletionTestSupport.java

```diff
@@ -114,7 +113,7 @@ public void run() {
 
         try {
             assertCompletionItemNames(expectedItemsNames, items, type);
-        } catch (AssertionFailedError e) {
+        } catch (AssertionError e) {
```

Comment:
Wouldn't it be easier to create this output in `assertCompletionItemNames`, add a sensible info in front of it and use it as the message of the delegated `assert*` calls there?

Reply:
the problem is that this is a public utility method with several usages. I didn't want to rewrite too much and make this PR even larger. Simply catching `AssertionError` seemed like the easiest solution.

It would have been dangerous if the utility method would call into NB code which has java assert statements and the test would expect a junit assert failure. If NB code would fail for the wrong reason, the test would pass instead of fail. But this here should be safe.

Reply:
I won't go into discussion over this and feel free to ignore this.